### ISSUE TITLE
Receiving external packets with laravel-websockets

### ISSUE THREAD

**Issue by essangjesse** [beyondcode/laravel-websockets#272]
I have an application that receives data from a sensor and I have configured it to send the data to 127.0.0.1:6000 which is where I have my laravel websocket server running. I want laravel websockets to listen to and receive packets from the app and then be able to receive those packets on my laravel application.

![sscom](https://user-images.githubusercontent.com/38280801/67093296-36a41200-f1a9-11e9-9528-6122e8e0c15b.png)

**Comment by rennokki** [comment]
Please elaborate a bit the issue, I cannot seem to understand what's actually not working. Showing a screenshot of a software won't help.

**Comment by essangjesse** [comment]
I wanted to send data from a sensor to my Laravel app. The sensor was configured to send data to the system's USB port. 

The goal was to send the data as packets from the USB port to the Laravel port I created using your library. 

Didn't get a response here on time, so kept doing research and discovered React-PHP. Used that to open up a port on my local development server and used a third-party USB-to-Com-port software to send the packets from the usb port to the com port I created, which I could then process from within Laravel.
